src/pages/categories/categories-add/categories-add.js

```diff
@@ -517,14 +519,26 @@ const CategoriesAdd = ({ id, editMode }) => {
           </Grid>
         </Grid>
       </FormControl>
-      <Button
-        variant='contained'
-        color='primary'
-        className={classes.saveBtn}
-        onClick={editMode ? handleCategoryEdit : handleCategorySave}
-      >
-        {config.buttonTitles.titleGenerator(editMode, isMain)}
-      </Button>
+
+      <div className={classes.controlsBlock}>
+        <Link to={config.routes.pathToCategories}>
+          <StandardButton
+              id='back'
+              title='Назад'
+              variant='outlined'
+              onClickHandler={() => {}}
+              data-cy='back-btn'
+          />
+        </Link>
```

Comment:
Please move title to config, and set link to onclickHandler

Reply:
done